app/serializers/question_answer_comment_serializer.rb

```diff
@@ -25,7 +25,7 @@ def user_voted
     if @comments_user_voted
       @comments_user_voted[object.id]
     else
-      object.votes.exists?(user: scope.user)
+      !!(scope && object.votes.exists?(user: scope.user))
```

Comment:
This `!!` syntax always feels bad to me...can't we just do `scope.present? && object.votes.exists?(user: scope.user)`

Reply:
Good point :+1: